.github/dependabot.yml

```diff
@@ -4,12 +4,39 @@ updates:
     directory: /
     schedule:
       interval: monthly
+    groups:
+      github-actions:
+        patterns:
+          - "*"
   - package-ecosystem: docker
     directory: /
     schedule:
       interval: monthly
+    groups:
+      docker:
+        patterns:
+          - "*"
   - package-ecosystem: gomod
     open-pull-requests-limit: 10
     directory: /
     schedule:
       interval: monthly
+    groups:
+      aws:
+        patterns:
+          - "github.com/aws/*"
+      docker:
+        patterns:
+          - "github.com/docker/*"
+          - "github.com/moby/*"
+      testcontainers:
+        patterns:
+          - "github.com/testcontainers/*"
+      common:
+        patterns:
+          - "*"
+        exclude-patterns:
+          - "github.com/aws/*"
+          - "github.com/testcontainers/*"
+          - "github.com/docker/*"
+          - "github.com/moby/*"
```

Comment:
Do we need these exclude-patterns? The document says as follows.

>Dependabot creates groups in the order they appear in your dependabot.yml file. If a dependency update could belong to more than one group, it is only assigned to the first group it matches with.

Reply:
`dependabot` added `aws` deps to `common` PR.
But looks like it was bug (perhaps because i very often updated config file 😄 )

I removed `exclude-patterns` and recheck.
It works as expected.

Thanks for bringing this to my attention.
Updated in https://github.com/aquasecurity/trivy/pull/6734/commits/3a7aee70a7852d69340cd29a9bb1e83c8463a6db